File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p02772/p02772_num8_parsed.py
n = int(input())
numbers = list(map(int, input().split()))

allowed = True
for num in numbers:
    if num % 2 == 0 and (num % 3 != 0 and num % 5 != 0):
        allowed = False
        break

if allowed:
    print("APPROVED")
else:
    print("DENIED")

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_cpp/p02772/p02772_num8_parsed.cpp
#include <iostream>
#include <vector>

using namespace std;

int main() {
    int N;
    cin >> N;

    vector<int> A(N);
    for (int i = 0; i < N; i++) {
        cin >> A[i];
    }

    for (int i = 0; i < N; i++) {
        if (A[i] % 2 == 0 && (A[i] % 3 != 0 && A[i] % 5 != 0)) {
            cout << "DENIED" << endl;
            return 0;
        }
    }

    cout << "APPROVED" << endl;

    return 0;
}